src/Cosmos/JsonSerialization.cpp

```diff
@@ -145,7 +144,7 @@ json messageWasmTerraTransfer(const Proto::Message_WasmTerraExecuteContractTrans
             {
                 {"sender", msg.sender_address()},
                 {"contract", msg.contract_address()},
-                {"execute_msg", wasmTerraExecuteTransferPayload(msg)},
+                {"execute_msg", protobuf::wasmTerraExecuteTransferPayload(msg)},
```

Comment:
Our namespace is usually upper case
```suggestion
                {"execute_msg", Protobuf::wasmTerraExecuteTransferPayload(msg)},
```